From 4d3ed123e65c757f42dbd84fa49b29be27ae4173 Mon Sep 17 00:00:00 2001
From: KO Myung-Hun <komh@chollian.net>
Date: Fri, 26 Jan 2018 15:00:35 +0900
Subject: [PATCH 015/120] Revert "qt: display directx rendering options only
 when vout is set to directx"

This reverts commit 7526658131dc6b222f156593b2ad5e2c81737f72.
---
 .../gui/qt/components/simple_preferences.cpp   | 18 ------------------
 .../gui/qt/components/simple_preferences.hpp   |  1 -
 2 files changed, 19 deletions(-)

diff --git a/modules/gui/qt/components/simple_preferences.cpp b/modules/gui/qt/components/simple_preferences.cpp
index f17c5a9534..f83c27466a 100644
--- a/modules/gui/qt/components/simple_preferences.cpp
+++ b/modules/gui/qt/components/simple_preferences.cpp
@@ -380,10 +380,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
             CONFIG_BOOL( "video-deco", windowDecorations );
             CONFIG_GENERIC( "vout", StringList, ui.voutLabel, outputModule );
 
-            CONNECT( ui.outputModule, currentIndexChanged( int ),
-                     this, updateVideoOptions( int ) );
-            optionWidgets["videoOutCoB"] = ui.outputModule;
-
             optionWidgets["fullscreenScreenB"] = ui.fullscreenScreenBox;
             ui.fullscreenScreenBox->addItem( qtr("Automatic"), -1 );
             int i_screenCount = 0;
@@ -406,7 +402,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
             CONFIG_BOOL( "directx-overlay", overlay );
             CONFIG_BOOL( "directx-hw-yuv", hwYUVBox );
             CONNECT( ui.overlay, toggled( bool ), ui.hwYUVBox, setEnabled( bool ) );
-            optionWidgets["directxVideoB"] = ui.directXBox;
 #else
             ui.directXBox->setVisible( false );
 #endif
@@ -430,8 +425,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
                             snapshotsSequentialNumbering );
             CONFIG_GENERIC( "snapshot-format", StringList, ui.arLabel,
                             snapshotsFormat );
-
-            updateVideoOptions( ui.outputModule->currentIndex() );
          END_SPREFS_CAT;
 
         /******************************
@@ -988,17 +981,6 @@ SPrefsPanel::SPrefsPanel( intf_thread_t *_p_intf, QWidget *_parent,
 #undef CONFIG_BOOL
 }
 
-void SPrefsPanel::updateVideoOptions( int number )
-{
-    QString value = qobject_cast<QComboBox *>(optionWidgets["videoOutCoB"])
-                                            ->itemData( number ).toString();
-#ifdef _WIN32
-    if( optionWidgets["directxVideoB"] ) {
-        optionWidgets["directxVideoB"]->setVisible( ( value == "directdraw" ) );
-    }
-#endif
-}
-
 
 void SPrefsPanel::updateAudioOptions( int number)
 {
diff --git a/modules/gui/qt/components/simple_preferences.hpp b/modules/gui/qt/components/simple_preferences.hpp
index 856b88ba97..6390b21efd 100644
--- a/modules/gui/qt/components/simple_preferences.hpp
+++ b/modules/gui/qt/components/simple_preferences.hpp
@@ -117,7 +117,6 @@ private:
 /* Display only the options for the selected audio output */
 private slots:
     void lastfm_Changed( int );
-    void updateVideoOptions( int );
     void updateAudioOptions( int );
     void updateAudioVolume( int );
     void langChanged( int );
-- 
2.50.1

